Resources/Views/pages/about.leaf

```diff
@@ -0,0 +1,106 @@
+#extend("layout/base")
+
+#export("content") {
+
+    <div class="container">
+        <div class="first-content m-b-30">
+
+            <div class="row">
+                <div class="col-xs-12">
+                    <h1 class="text-center fc-orange">About</h1>
+                    
+                    <h3>The Conference</h3>
+                    <h4 class="f-weight-sm">
+                        ServerSide.swift is a framework-independent conference, where we will learn and share on a number of different related topics.
+                        The conference is aimed at being a non-profit conference and solely run for the love of server-side Swift. This space is still new and we hope to be able to contribute to the growth of ServerSide.swift by facilitating a conference where people can meet and collaborate. We have all been following and taking part in the server-side Swift movement and after running meetups in Berlin and London, we believe that the community is ready for a conference dedicated to server-side Swift.
+                    </h4>
+                    <h4 class="f-weight-sm">
+                        We want people to feel safe and welcome at the conference and we expect all attendees to respect our <a href="/code-of-conduct">Code of Conduct</a>.
+                    </h4>
+
+                    <h3>The Team</h3>
```

Comment:
I would wrap it into a `<div class="col-xs-12"></div>` and put it within the next `<div class="row">` right before all the `<div class="col-xs-12 col-sm-4 col-md-4">` columns of our images come. 😊